rasa_nlu/data_router.py

```diff
@@ -210,7 +235,9 @@ def start_train_process(self, data, config_values):
             _config[key] = val
         _config["data"] = f.name
         train_config = RasaNLUConfig(cmdline_args=_config)
-        process = multiprocessing.Process(target=do_train, args=(train_config, self.component_builder))
-        self._add_train_proc(process)
```

Comment:
Ok I'll track endpoints status then. The difference with the agents endpoints would be that [looking for models](https://github.com/PHLF/rasa_nlu/blob/6905895a3216ed6fd05b4a5f228cf3a1c7e8fcae/rasa_nlu/data_router.py#L109) may create new endpoints from old model files.